app/assets/stylesheets/custom.css.scss

```diff
@@ -227,13 +227,18 @@ p.copyright {
     opacity: 0;
     filter: alpha(opacity=0);
 }
+
 .q-width{
   margin: auto;
   width: 40%;
   min-width: 300px;
 }
-.row .btn{
-  margin-bottom: 1em;
+
+.btn-margin{

```

Comment:
@OrestF  I think you should try one more time
there couple mistakes
- main mistake - it's name - btn-margin - very bad name - it looks like general class but in fact do nothing special  (should be specific to topic create button or create button)
-  aligning - whole css block looks terrible and it's hard to read it - make it fancy
-  space before ```{```
  -